<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>566</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>690</width>
    <height>566</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SQL editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/icons.qrc">
    <normaloff>:/icons/database.ico</normaloff>:/icons/database.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox_sql_connect">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>330</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::ActionsContextMenu</enum>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="title">
         <string>Connect to server</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <widget class="QGroupBox" name="groupBox_authentication">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>114</y>
           <width>231</width>
           <height>141</height>
          </rect>
         </property>
         <property name="title">
          <string>Authentication</string>
         </property>
         <widget class="QLineEdit" name="lineEdit_login">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>54</y>
            <width>131</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>sa</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_password">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>84</y>
            <width>131</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>mahdi1376+sa</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>54</y>
            <width>61</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>login:</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>80</y>
            <width>71</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>password:</string>
          </property>
         </widget>
        </widget>
        <widget class="QPushButton" name="button_connect">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>300</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Connect</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>59</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>server:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_server_address">
         <property name="geometry">
          <rect>
           <x>102</x>
           <y>59</y>
           <width>141</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>127.0.0.1</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_database_name">
         <property name="geometry">
          <rect>
           <x>102</x>
           <y>270</y>
           <width>141</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Hospital_Switching_System</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>270</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>database:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>89</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>port:</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_server_port">
         <property name="geometry">
          <rect>
           <x>102</x>
           <y>89</y>
           <width>61</width>
           <height>22</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Default: MySQL 3306; MSSQL 1433</string>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
         <property name="value">
          <number>1443</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>29</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>driver:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_driver">
         <property name="geometry">
          <rect>
           <x>102</x>
           <y>29</y>
           <width>141</width>
           <height>20</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>For ODBC/MSSQL: &quot;SQL Server&quot; (Windows), &quot;ODBC Driver 13 for SQL Server&quot; (unix), etc.</string>
         </property>
         <property name="text">
          <string>ODBC Driver 17 for SQL Server</string>
         </property>
        </widget>
        <widget class="QGroupBox" name="groupBox_operation">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>340</y>
           <width>250</width>
           <height>90</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>operation</string>
         </property>
         <widget class="QRadioButton" name="radioButton_crud">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>30</y>
            <width>112</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>CRUD</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="radioButton_search">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>60</y>
            <width>112</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox_crudMethod">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>430</y>
           <width>251</width>
           <height>91</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>91</height>
          </size>
         </property>
         <property name="title">
          <string>CRUD Method</string>
         </property>
         <widget class="QRadioButton" name="radioButton_create">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>112</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Create</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="radioButton_delete">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>60</y>
            <width>112</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="radioButton_Update">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>30</y>
            <width>112</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="radioButton_read">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>112</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Read</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_database_browser">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
      <property name="title">
       <string>Database browser</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="5" column="0">
        <widget class="QLabel" name="label_where">
         <property name="text">
          <string>Where</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>table:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QComboBox" name="comboBox_table_name">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>0</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>130</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="lineEdit_search">
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLineEdit" name="lineEdit_where">
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="5">
        <widget class="QTableView" name="tableView_database_table">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="tabKeyNavigation">
          <bool>false</bool>
         </property>
         <property name="dragDropOverwriteMode">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="button_show_table">
         <property name="text">
          <string>Show</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_field">
         <property name="text">
          <string>Field</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_search">
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="3">
        <widget class="QComboBox" name="comboBox_field_name">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>0</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>130</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_driver</tabstop>
  <tabstop>lineEdit_server_address</tabstop>
  <tabstop>spinBox_server_port</tabstop>
  <tabstop>lineEdit_login</tabstop>
  <tabstop>lineEdit_password</tabstop>
  <tabstop>lineEdit_database_name</tabstop>
  <tabstop>button_connect</tabstop>
  <tabstop>comboBox_table_name</tabstop>
  <tabstop>tableView_database_table</tabstop>
 </tabstops>
 <resources>
  <include location="../res/icons.qrc"/>
 </resources>
 <connections/>
</ui>
